KMB/make_KMB_info.py

```diff
@@ -171,6 +174,27 @@ def get_photographer_mapping(self, photographer_page):
                 qid, photographer_props, self.photographer_cache)
         return photographers
 
+    def load_wikidata_bbr_fmis_commonscat(self):
+        """
+        Load all bbr/fmis entries in Wikidata and add any commonscats.
+
+        Overrides any mappings found in heritage.
+        """
+        query_props = {'P373': 'commonscat'}
+        mapped_data = self.mappings['commonscat']
+        data = KMBInfo.query_to_lookup(
+            KMBInfo.build_query('P1260', optional_props=query_props.keys()),
+            props=query_props)
+
+        for k, v in data.iteritems():
+            if v.get('commonscat'):
+                entry = {'wd': v.get('wd'), 'cat': v.get('commonscat')}
+                prefix, _, idno = k.rpartition('/')
```

Comment:
So now I get this:
```python
k = "raa/fmi/html/123455"

prefix, _, idno = k.rpartition('/')
if prefix.endswith('/html'):
    prefix = prefix[len('/html'):]

print(prefix)
```

    mi/html


Move the colon to the other side of the slicer?